sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -559,6 +560,11 @@ columnConstraint
     | INDEX USING method=ident withProperties?                                       #columnIndexConstraint
     | INDEX OFF                                                                      #columnIndexOff
     | STORAGE withProperties                                                         #columnStorageDefinition
+    | checkConstraint                                                                #columnCheckConstraint
```

Comment:
fyi: you can use the same rule name instead of `columnCheckConstraint` and `tableCheckConstraint` , then you do not have to implement two visitor methods.